.github/workflows/terraform-lint.yml

```diff
@@ -11,6 +11,10 @@ on:
         description: 'The directory upon which to lint Terraform configurations.'
         type: 'string'
         required: true
+    secrets:
+      gh_token:
```

Comment:
Can we name this github_token please :)

Reply:
Ah i forgot i tried this before.....github wont let you:

https://github.com/abcxyz/pkg/actions/runs/4314396435

```
secret name `github_token` within `workflow_call` can not be used since it would collide with system reserved name
```